content/post/kubedb-v2024.4.27/index.md

```diff
@@ -0,0 +1,415 @@
+
+To learn more, have a look [here](https://github.com/appscode/alerts/tree/master/charts)
+
+## Kafka
+
+`Pod Disruption Budget (PDB)` support has been added for Kafka. A PDB limits the number of Pods of a replicated application that are down simultaneously from voluntary disruptions. For example, a quorum-based application like Kafka would like to ensure that the number of replicas running is never brought below the number needed for a KRaft quorum.
+
+### Kafka-Connect
+Support for `JDBC Connector (Source and Sink)` have been added for KubeDB managed Kafka connect clusters. This connector will allow you to exchange data between relational databases and Kafka. The JDBC source connector allows you to import data from any relational database with a JDBC driver into Kafka topics. `Pod Disruption Budget (PDB)` support also has been added for Kafka Connect.
+
+## Microsoft SQL Server
+
+We are thrilled to announce that KubeDB now extends its support to `Microsoft SQL Server`, one of the most popular relational database management systems (RDBMS) in the world. With this addition, KubeDB users can now seamlessly provision and manage SQL Server instances directly within their Kubernetes clusters.
+
+This release includes support for `Provisioning SQL Server Availability Group` and S`tandalone SQL Server` instance
+Utilize SQL Server's high availability features by deploying instances in availability group mode. KubeDB leverages the `Raft Consensus Algorithm for cluster coordination`, enabling automatic leader election and failover decisions. `Quorum support` ensures the reliability and fault tolerance of your SQL Server deployments.
+Deploy SQL Server instances in standalone mode for simple, single-node configurations.
+
+Here’s a sample YAML to provision the Availability Group cluster. To generate the certificate used for internal endpoint authentication of availability group replicas, an Issuer named `mssqlserver-ca-issuer` need to be created prior to deploying the following manifest.
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: MSSQLServer
+metadata:
+  name: sqlserver-ag
+  namespace: demo
+spec:
+  version: "2022-cu12"
+  replicas: 3
+  topology:
+    mode: AvailabilityGroup
+    availabilityGroup:
+      databases:
+        - AgDB1
+        - AgDB2
+  internalAuth:
+    endpointCert:
+      issuerRef:
+        apiGroup: cert-manager.io
+        name: mssqlserver-ca-issuer
+        kind: Issuer
+  storageType: Durable
+  storage:
+    storageClassName: "standard"
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+  terminationPolicy: Delete
+```
+
+#### supported versions:
+`2022-CU12-ubuntu-22.04`
+
+## MongoDB
+
+- KubeDB support for two new MongoDB versions `5.0.26` & `7.0.8` have been introduced in this release.
+
+- RestoreSession still running even after a failure bug fixes for MongoDB Restic backup and restore support.
```

Comment:
Bug fix on updating the RestoreSession status:  After a failed restore process with KubeStash Restic support, the RestoreSession status was not updated correctly. That has been fixed in this release.